torch/_torch_docs.py

```diff
@@ -7385,6 +7385,35 @@ def merge_dicts(*dicts):
             [3, 2, 1, 0]])
 """.format(**common_args))
 
+add_docstr(torch.msort,
+           r"""
+msort(input, *, out=None) -> Tensor
+
+Sorts the elements of the :attr:`input` tensor along its first dimension
+in ascending order by value.
+
+.. note:: `torch.msort(a)` is equivalent to `torch.sort(t, dim=0)[0]`.
```

Comment:
fyi you can see the rendered docs here: https://9172902-65600975-gh.circle-artifacts.com/0/docs/generated/torch.msort.html

"torch.msort(a)" -> "torch.msort(t)" to be consistent with the input variable name to "torch.sort".

Reply:
Addressed!